client/js/controllers/post_list_controller.js

```diff
@@ -44,13 +45,24 @@ class PostListController {
             enableSafety: api.safetyEnabled(),
             canBulkEditTags: api.hasPrivilege("posts:bulk-edit:tags"),
             canBulkEditSafety: api.hasPrivilege("posts:bulk-edit:safety"),
+            canBulkDelete: api.hasPrivilege("posts:bulk-edit:delete"),
             bulkEdit: {
                 tags: this._bulkEditTags,
             },
         });
         this._headerView.addEventListener("navigate", (e) =>
             this._evtNavigate(e)
         );
+        // This doesn't feel like the best solution
+        this._headerView._bulkDeleteEditor.addEventListener(
```

Comment:
This isn't pretty, because we call addEventListener on a (theoretically speaking) private field.